package/kernel/mt76/patches/000-wifi-mt76-support-per-band-MAC-addresses-from-OF-chi.patch

```diff
@@ -0,0 +1,60 @@
+From 33cf7852a90b638f6a481ae8d860d2b75212e906 Mon Sep 17 00:00:00 2001
+From: Daniel Golle <daniel@makrotopia.org>
+Date: Thu, 13 Jul 2023 01:22:43 +0100
+Subject: [PATCH] wifi: mt76: support per-band MAC addresses from OF child
+ nodes
+
+With dual-band-dual-congruent front-ends which appear as two independent
+radios it is desirable to assign a per-band MAC address from device-tree,
+eg. using nvmem-cells.
+Support specifying MAC-address related properties in band-specific child
+nodes, e.g.
+        mt7915@0,0 {
+                reg = <0x0000 0 0 0 0>;
+                #addr-cells = <1>;
```

Comment:
@dangowrt I tested your mt76 MAC address patch, it works well on mt7915d. There is a small issue on commit message. `#addr-cells` should be changed to `#address-cells`. And it would be better to add these usages to dt-bindings document.